### PR TITLE
Add timed commands

### PR DESCRIPTION
**Pull request by Mike-HomeCU**
Adding MoveTimed and SpinTimed commands. Also added Drivetrain set
and stop.

### CODE REVIEW

File: src/main/java/frc/robot/commands/MoveTimed.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+package frc.robot.commands;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+public class MoveTimed extends CommandBase {
+  private Drivetrain drivetrain;
```
Comment (warrenlp) [inline src/main/java/frc/robot/commands/MoveTimed.java:16]: Within the new Command-based infrastructure, the RobotContainer class owns the Subsystems and all requests to "do" something to the robot should go through the RobotContainer which then asks the Subsystem to perform the specific action. This should probably be abstracted away from the Command in the sense that the Command is only talking to the RobotContainer and not having the RobotContainer pass the Subsystem through to the Command.

File: src/main/java/frc/robot/commands/MoveTimed.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+package frc.robot.commands;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+public class MoveTimed extends CommandBase {
```
Comment (warrenlp) [inline src/main/java/frc/robot/commands/MoveTimed.java]: This class should probably be of type "WaitCommand" which has the wpilib Timer built into it already.

File: src/main/java/frc/robot/commands/MoveTimed.java (hunk 1 above)
Comment (devinvs) [reply src/main/java/frc/robot/commands/MoveTimed.java:16]: When I looKed into how to interface with the subsystems, the examples I found passed the subsystems as a parameter. See https://github.com/wpilibsuite/allwpilib/tree/master/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/gearsbot

Comment (warrenlp) [reply src/main/java/frc/robot/commands/MoveTimed.java:16]: Yep, you beat me to it. The commands now have joint ownership of the subsystems (along with the RobotContainer) on which they operate and therefore do not have to go though the RobotContainer to access the Subsystem. I actually think that this is cleaner and retract my comment from earlier.

**Review by devinvs — APPROVED** [review]

**Pull request merged**